lib/fog/aws/requests/data_pipeline/activate_pipeline.rb

```diff
@@ -14,16 +14,20 @@ def activate_pipeline(id)
 
           response = request({
             :body => Fog::JSON.encode(params),
-            :headers => { 'X-Amz-Target' => 'DataPipeline.ActivatePipeline' },
+            :headers => { 'X-Amz-Target' => 'DataPipeline.ActivatePipeline' }
           })
-
-          Fog::JSON.decode(response.body)
```

Comment:
Definitely brings it in line with the rest of the services.  Without making that change, it makes it impossible to assert on things that return an empty body, but different status codes.